api/test/e2e/server_info_test.go

```diff
@@ -121,25 +112,14 @@ func TestServerInfo_Get_OmitEmptyValue(t *testing.T) {
 func TestServerInfo_List_OmitEmptyValue(t *testing.T) {
 	testCases := []HttpTestCase{
 		{
-			Desc:           "list all server info",
-			Object:         ManagerApiExpect(t),
-			Path:           "/apisix/admin/server_info",
-			Method:         http.MethodGet,
-			Headers:        map[string]string{"Authorization": token},
-			ExpectStatus:   http.StatusOK,
-			ExpectBody:     "\"total_size\":2",
-			UnexpectedBody: "\"create_time\":",
-		},
-		{
-			Desc:           "list server info with hostname",
-			Object:         ManagerApiExpect(t),
-			Path:           "/apisix/admin/server_info",
-			Query:          "hostname=apisix_",
-			Method:         http.MethodGet,
-			Headers:        map[string]string{"Authorization": token},
-			ExpectStatus:   http.StatusOK,
-			ExpectBody:     "\"total_size\":2",
-			UnexpectedBody: "\"update_time\":",
```

Comment:
Hi, why delete this test?
This test is for `list server info with hostname`.

Reply:
@starsz oh, sorry, my bad. Will restore it soon.

Reply:
@starsz PR renewed with restoring the mistakenly removed code, much appreciated for your reviewing.